text/processing.py

```diff
@@ -202,3 +204,28 @@ def run(self):
             self.lines = int(length * self.ratio)
 
         self.sh("zcat -f {text_file} | tail -n {num_lines} | gzip > {out}")
+
+
+class DiffJob(Job):
+    """
+    Return the set difference of two text files, where one line is one element.
+    """
+
+    def __init__(self, file1, file2):
```

Comment:
pls. add a doc string for file1 and file2
e.g. so that the user knows the diff is file1 - file2 and not the other way round.

Maybe there is better names to make sure which file is which (e.g. minuend and subtrahend - ok, this is set difference not subtraction so not entirely correct nomenclature)... But we can also leave it like this.